src/parser/transformer.cpp

```diff
@@ -1050,20 +1050,24 @@ std::string Transformer::transformPrimaryKey(CypherParser::KU_CreateNodeConstrai
 }
 
 std::unique_ptr<Statement> Transformer::transformCopyCSV(CypherParser::KU_CopyCSVContext& ctx) {
+    auto direction = ctx.FROM() ? common::CopyDescription::Direction::FROM :
+                                  common::CopyDescription::Direction::TO;
     auto filePaths = transformFilePaths(ctx.kU_FilePaths()->StringLiteral());
     auto tableName = transformSchemaName(*ctx.oC_SchemaName());
     auto parsingOptions = ctx.kU_ParsingOptions() ?
                               transformParsingOptions(*ctx.kU_ParsingOptions()) :
                               std::unordered_map<std::string, std::unique_ptr<ParsedExpression>>();
-    return std::make_unique<Copy>(std::move(filePaths), std::move(tableName),
+    return std::make_unique<Copy>(direction, std::move(filePaths), std::move(tableName),
         std::move(parsingOptions), common::CopyDescription::FileType::UNKNOWN);
 }
 
 std::unique_ptr<Statement> Transformer::transformCopyNPY(CypherParser::KU_CopyNPYContext& ctx) {
+    auto direction = ctx.FROM() ? common::CopyDescription::Direction::FROM :
```

Comment:
You probably don't need to check direction because the grammar only allows COPY FROM. So give Direction::FROM directly to the constructor